src/grpcbox_acceptor.erl

```diff
@@ -16,7 +16,8 @@ acceptor_continue(_PeerName, Socket, {ssl, _MRef, PoolName, ServerOpts, Chatterb
     case ssl:negotiated_protocol(AcceptSocket) of
         {ok, <<"h2">>} ->
             MaxConns = maps:get(max_connections, ServerOpts, unlimited),
-            case connection_allowed(PoolName, MaxConns) of
+            {ok, {PeerName, _PeerPort}} = ssl:peername(AcceptSocket),
```

Comment:
we don't use ssl for grpc connections, is that going to be a problem here? should we use the `socket:peername/1` instead?

Reply:
there is a diff handler for tcp connection